### PR TITLE
Feature/firebase auth

### PR DESCRIPTION
**Pull request by m1chalkubiak**
Launch screen/start menu + Google auth using Firebase

### CODE REVIEW

File: webapp/src/shared/components/launcher/background/background.component.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { Container, Sprite, Texture } from 'pixi.js';
+
+class Background extends Container {
+  constructor({ image }) {
+    super();
+    this.texture = Texture.from(image);
+    this.image = new Sprite(this.texture);
+    this.image.width = 1;
```
Comment (mkmieciak) [inline webapp/src/shared/components/launcher/background/background.component.js]: why 1?

File: webapp/src/shared/components/launcher/background/background.component.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { Container, Sprite, Texture } from 'pixi.js';
+
+class Background extends Container {
```
Comment (mkmieciak) [inline webapp/src/shared/components/launcher/background/background.component.js]: I would not want to use extends becuase it is very easy to overwrite the original prop which you don't know

File: webapp/src/routes/home/game/index.js [hunk 3]
```diff
@@ -18,9 +21,30 @@ export class Game {
       width: this.width,
       height: this.height,
     });
+    this._loginViaGoogle = loginViaGoogle;
 
     this.htmlElement.append(this._app.renderer.view);
 
+    ifElse(
```
[the remark this answers is not in the document]
Comment (mkmieciak) [reply webapp/src/routes/home/game/index.js]: or in next PR

File: webapp/src/shared/components/launcher/background/background.component.js (hunk 2 above)
Comment (m1chalkubiak) [reply webapp/src/shared/components/launcher/background/background.component.js]: Good point, I'm going to refactor this.

File: webapp/src/shared/components/launcher/background/background.component.js (hunk 1 above)
Comment (m1chalkubiak) [reply webapp/src/shared/components/launcher/background/background.component.js]: Well, this value does not impact to anything, but lack of it finishes with not displayed image at all. 😕

**Pull request merged**